### PR TITLE
XIVY-10526 Use "ivy-elasticsearch" as log file prefix instead of cluster name

### PR DESCRIPTION
**Pull request by jackra1n**
image

### CODE REVIEW

File: log4j2.properties [hunk 1]
```diff
@@ -1,3 +1,5 @@
+property.log.prefix = ivy-elasticsearch
```
Comment (alexsuter) [inline log4j2.properties:1]: I would use "elasticsearch" as prefix. Don't see the need for ivy?!

File: log4j2.properties [hunk 2]
```diff
@@ -24,18 +26,18 @@ appender.console.layout.pattern = [%d{ISO8601}][%-5p][%-25c{1.}] [%node_name]%ma
...
 ######## Server -  old style pattern ###########
 appender.rolling_old.type = RollingFile
 appender.rolling_old.name = rolling_old
-appender.rolling_old.fileName = ${sys:es.logs.base_path}${sys:file.separator}${sys:es.logs.cluster_name}.log
+appender.rolling_old.fileName = ${sys:es.logs.base_path}${sys:file.separator}${log.prefix}.log
 appender.rolling_old.layout.type = PatternLayout
 appender.rolling_old.layout.pattern = [%d{ISO8601}][%-5p][%-25c{1.}] [%node_name]%marker %m%n
 
-appender.rolling_old.filePattern = ${sys:es.logs.base_path}${sys:file.separator}${sys:es.logs.cluster_name}-%d{yyyy-MM-dd}-%i.log.gz
+appender.rolling_old.filePattern = ${sys:es.logs.base_path}${sys:file.separator}${log.prefix}-%d{yyyy-MM-dd}-%i.log.gz
```
Comment (alexsuter) [inline log4j2.properties:40]: is the date format the same as for Axon Ivy Engin elog files.
And what is -%i for?

Comment (jackra1n) [reply log4j2.properties:40]: @alexsuter  %i is a integer counter

Comment (alexsuter) [reply log4j2.properties:40]: we don't need it?

**Review by alexsuter — APPROVED** [review]
nice!

**Pull request merged**